### PR TITLE
adding a note about the scored annotation

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**

### CODE REVIEW

File: content/en/docs/Policy Reports/_index.md [hunk 1]
```diff
@@ -50,6 +50,10 @@ kubectl get clusterpolicyreport -A
 `cpolr` is the short name for `clusterpolicyreport`.
 {{% /alert %}}
 
+{{% alert title="Note" color="info" %}}
```
Comment (realshuting) [inline content/en/docs/Policy Reports/_index.md:53]: Let's clarify the default is set to true.

**Review by realshuting — APPROVED** [review]

**Comment by realshuting** [comment]
@RinkiyaKeDad - can you resolve conflicts?

**Comment by RinkiyaKeDad** [comment]
@realshuting sorry I somehow missed the notification for this. I've resolved the conflict. We should be good to merge now.

**Review by realshuting — APPROVED** [review]

**Comment by RinkiyaKeDad** [comment]
@realshuting can you please merge this, thanks!

**Pull request merged**